Source/JavaScriptCore/runtime/VM.h

```diff
@@ -1154,6 +1086,7 @@ class VM : public ThreadSafeRefCountedWithSuppressingSaferCPPChecking<VM>, publi
     Lock m_loopHintExecutionCountLock;
     UncheckedKeyHashMap<const JSInstruction*, std::pair<unsigned, std::unique_ptr<uintptr_t>>> m_loopHintExecutionCounts;
 
+    Ref<MicrotaskQueue> m_defaultMicrotaskQueue;
```

Comment:
What's the difference between `m_defaultMicrotaskQueue` and `m_microtaskQueues`?  What is the meaning of "default" in this case?

Reply:
`default`-initialized queue and used when no specific MicrotaskQueue is instantiated. This is just keeping the current behavior for JSC framework etc. But later, we will add a feature having multiple MicrotaskQueue created at runtime for WebCore, then this is not used in the main thread (while WebCore Worklet still uses this default queue too).